src/controllers/controller.line.js

```diff
@@ -117,18 +117,18 @@ module.exports = DatasetController.extend({
 		var me = this;
 		var meta = me.getMeta();
 		var custom = point.custom || {};
-		var dataset = me.getDataset();
 		var datasetIndex = me.index;
-		var value = dataset.data[index];
 		var xScale = me._xScale;
 		var yScale = me._yScale;
 		var lineModel = meta.dataset._model;
+		var xvalue = me._getParsedValue(index, xScale);
+		var yvalue = me._getParsedValue(index, yScale);
```

Comment:
Inline this so that you don't need to make an extra method call when `reset` is true